modules/customize-certificates-add-service-serving-apiservice.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * authentication/certificates/service-serving-certificate.adoc
+
+[id="add-service-certificate-apiservice_{context}"]
+= Add the service CA bundle to an APIService
+
+You can annotate an APIService with `service.beta.openshift.io/inject-cabundle=true` to have its `caBundle` field populated with the service CA bundle. This allows the Kubernetes API server to validate the service CA certificate used to secure the targeted endpoint.
+
+.Procedure
+
+. Annotate the APIService with `service.beta.openshift.io/inject-cabundle=true`:
++
+----
+$ oc annotate apiservice <apiservice-name> \//<1>
+     service.beta.openshift.io/inject-cabundle=true
+----
+<1> Replace `<apiservice-name>` with the name of the APIService to annotate.
++
+For example, use the following command to annotate the APIService `foo`:
++
+----
+$ oc annotate apiservice foo service.beta.openshift.io/inject-cabundle=true
+----
+
+. View the APIService to ensure that the certificate has been generated:
++
+----
+$ oc get apiservice <apiservice-name> -o yaml
+----
++
+It appears as `caBundle` in the YAML output:
```

Comment:
s/It appears/It is displayed/
**appear** _v_
Do not use unless there is no better alternative. Possible alternatives include “is displayed” and “opens.”